src/calendar-heatmap.component.js

```diff
@@ -784,12 +810,21 @@ export class CalendarHeatmap extends Component {
     let month_data = this.props.data.filter((d) => {
       return start_of_month <= moment(d.date) && moment(d.date) < end_of_month;
     });
+
+    // Calculate max value of month in the dataset
     let max_value = max(month_data, (d) => {
       return max(d.summary, (d) => {
         return d.value;
       });
     });
 
+    // Calculate min value of month in the dataset
+    let min_value = min(month_data, (d) => {
+      return min(d.summary, (d) => {
+        return d.value;
```

Comment:
Here also try to use extent, you can use flatMap first to create an array of data